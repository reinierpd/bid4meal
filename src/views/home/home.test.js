/* eslint-env jest */
/* eslint-disable react/jsx-filename-extension */
import React from "react";
import { shallow } from "enzyme";
import Home from "./home";

describe("Home", () => {
  it("should render ok", () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
});
