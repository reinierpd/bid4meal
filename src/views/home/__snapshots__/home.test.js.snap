// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render ok 1`] = `
<ForwardRef(Bootstrap(Container))>
  <h1
    className="display-4"
  >
    My Booking
  </h1>
  <hr
    className="my-4"
  />
  <Booking />
</ForwardRef(Bootstrap(Container))>
`;
